services/ui-src/src/components/drawers/Drawer.tsx

```diff
@@ -64,7 +63,7 @@ interface Props {
   };
   drawerDisclosure: {
     isOpen: boolean;
-    onClose: Function;
+    onClose: any;
```

Comment:
was the change to `any` just to cleanup the `MouseEventHandler`?

Reply:
yeah we had been modifying the function call there, just reverted it though